File: CS/App_Code/BusinessObject.cs
using System;
using System.Web;
using System.Data;
namespace PubsClasses
{
	public class AuthorClass
	{
		private DataSet dsAuthors = new System.Data.DataSet("ds1");
		private string filePath = HttpContext.Current.Server.MapPath ("~/App_Data/Authors.xml");
		public AuthorClass()
		{
			dsAuthors.ReadXml(filePath, System.Data.XmlReadMode.ReadSchema);
		}

		public DataSet GetAuthors()
		{
			return dsAuthors;
		}
	}
}

File: VB/App_Code/BusinessObject.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Web
Imports System.Data
Namespace PubsClasses
	Public Class AuthorClass
		Private dsAuthors As DataSet = New System.Data.DataSet("ds1")
		Private filePath As String = HttpContext.Current.Server.MapPath ("~/App_Data/Authors.xml")
		Public Sub New()
			dsAuthors.ReadXml(filePath, System.Data.XmlReadMode.ReadSchema)
		End Sub

		Public Function GetAuthors() As DataSet
			Return dsAuthors
		End Function
	End Class
End Namespace
